src/components/DFR3Viewer.jsx

```diff
@@ -213,10 +215,15 @@ class DFR3Viewer extends React.Component {
 		});
 	}
 
-	// TODO set state inside component did up date is bad practice!!
+	// TODO set state inside componentDidUpdate is bad practice!!
```

Comment:
Is there an issue to fix this?

Reply:
I have issues to convert everything to use hook and this will go away